src/tjenesteomrader/areas/index.tsx

```diff
@@ -0,0 +1,40 @@
+import markdownit from 'markdown-it/lib';
+import { InferGetStaticPropsType } from 'next';
+import { getStaticProps } from 'pages/tjenesteomrader/[omrade]';
+import style from 'src/tjenesteomrader/shared/bottomText.module.css';
+import Divider from '../images/divider/divider';
+import Footer from '../images/footer/footer';
+import DigitalTjeneste from './DigitalTjeneste';
+import Strategi from './strategi';
+import Datadriv from './datadriv';
+import Kultur from './kultur';
+
+const Areas = ({ prop }: InferGetStaticPropsType<typeof getStaticProps>) => {
+  const md = markdownit({ linkify: true, html: true, typographer: true });
```

Comment:
Lurer på om dette kanskje like gjerne kunne blitt sendt med fra staticProps og ikke i render-funksjonen. Dersom du har det her kan det være en risiko med Client Side Rendering at den kalles flere ganger, som kan være en potensielt stor performance hit. Ville ha flyttet ut til der hvor du henter ut `await getMarkdownObject(filePath)`.

Også ville jeg ha eksplisitt typet komponenten her. Å type ting fra hva de trenger "bottom-up" kan ofte være mer eksplisitt og refactorerbart da du ikke knytter det opp for tett med den som sender dataen. Også er ting som `InferGetStaticPropsType<typeof getStaticProps>` vanskelig å umiddelbart se hva den trenger av informasjon.